### DISCUSSION TITLE
sometimes i get version conflict error when updating list item,how can i avoid this

### DISCUSSION THREAD

**Discussion by harshdamaniahd** [pnp/powershell#3378] [Q&A]
How can we avoid this error when updating list item in pnp powershell
System.Management.Automation.PSInvalidOperationException: Version conflict.
   at PnP.PowerShell.Commands.Base.PnPConnectedCmdlet.ProcessRecord() in c:\build\src\Commands\Base\PnPConnectedCmdlet.cs:line 98
   at PnP.PowerShell.Commands.PnPSharePointCmdlet.ProcessRecord() in c:\build\src\Commands\Base\PnPSharePointCmdlet.cs:line 120
   at System.Management.Automation.Cmdlet.DoProcessRecord()
   at System.Management.Automation.CommandProcessor.ProcessRecord()

**Comment by AndersRask** [comment]
Hi,

version conflict usually appears when client context version is higher than expected.
This is usually caused by doing several updates on the same item inside a loop.

Please post the code that sometimes thows this error, along with a tracelog (you enable trace log with` Set-PnPTraceLog -On -Logfile traceoutput.txt` )

Also please specify PowerShell and module version
```
Get-Module -Name "PnP.PowerShell" -ListAvailable
$psversiontable
```

**Reply by harshdamaniahd** [reply]
where should i add this Set-PnPTaceLog Command, in catch block ? i am using this  in azure runbook. Can you please let me know where to add it ?

**Reply by AndersRask** [reply]
You just fire it before running your script from a shell. It enables logging into the file specified with the log level specified
